nbs/01_config.ipynb

```diff
@@ -189,7 +189,16 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/hamel/opt/anaconda3/envs/nbdev/lib/python3.9/site-packages/ghapi/core.py:99: UserWarning: Neither GITHUB_TOKEN nor GITHUB_JWT_TOKEN found: running as unauthenticated\n",
```

Comment:
Can you define the env var and rerun so this doesn't appear in docs?

Reply:
This cell is hidden, BTW so doesn't appear in the docs!  but I can define it so it doesn't clutter the diff 😃

Reply:
@jph00 have some follow up questions here if you can take a look.  I didn't understand a few things.

Reply:
Mainly just didn't want it in the NB. Not a big deal though. (Mind you it's useful to have this set up in your .zshrc anyway!)

Reply:
Ok makes sense removed this thanks for explaining